ecmean.py

```diff
@@ -163,13 +165,15 @@ def load_yaml(infile):
     return cfg
 
 
-def make_input_filename(dr, var, expname, year1, year2, face):
+def make_input_filename(var, year1, year2, face, diag):
     """Create input filenames for the required variable and a given year"""
 
     filetype = face[var]['filetype']
-    fname = dr / 'output' / face[var]['component'] / \
-        f'{expname}_{filetype}_{year1}-{year2}.nc'
-    return str(fname)
+    filemask = face['model']['filetype'][filetype]['filename']
+    filename = Path(os.path.expandvars(filemask.format(expname=diag.expname, year1=year1, year2=year2)))
```

Comment:
This should be more general: for example, the filenames or the folder might include also the variable name (the replcamente with format is very cool!)

Reply:
That's right! My idea was to extend this in the future as needed (according to use case). I just added the suggested "var" to the formatting

Reply:
Great! I will test it today. Another thing that could be problematic - if we want to analyzed raw data - is to work with monthly files as IFS does. This would require - I guess - wildcards in the `filename` variable - which in principle is possible.

Reply:
I just added the capability to use a * in the filename mask in the interface file (this indeed required some change). If you want to test it you could:
1) modify config.yml to use the model with `name: EC-Earth4_mon`
this will use a corresponding sample interface which for now I have committed for testing (we will remove it later)
2) point to my `exp: $SCRATCH/../ccjh/ece4`
3) There I prepared a fake experimental output for two years (1990 and 1991) with separate monthly data, based on your BETA exp. You can test e.g. with:
`./global_mean.py META 1990 1991 -j 4`
The results are identical with the original analysis on the same years performed on BETA